### ISSUE TITLE
exceptions.AttributeError

### ISSUE THREAD

**Issue by mkayoh** [securestate/king-phisher#62]
Trying out the current dev build, I'm not sure if you're already aware of this but I keep getting this error when running the client. After the error message I get a message about invalid host key which is even weirder since neither client or the server has changed. It lists the server's RSA key in SHA256 and MD5 hashed versions. There is an option to check "I accept the risk" but checking it doesn't do anything and I can't accept and the only option left is reject. It occurred first when I switched the client from master to dev. Happens every time I try connecting with the dev client, after checking back to master branch everything seems to work normally.

Error Type: exceptions.AttributeError
Error Details: AttributeError('Handler signal_checkbutton_toggled not found',)
Error UID: 3ee77f56-5f97-4144-bf95-b24c56076472
RPC Error: N/A (Not a remote RPC error)
King Phisher Version: 1.2.0-beta
Platform Version: Linux: Ubuntu 14.04 trusty
Python Version: 2.7.6
Gtk Version: 3.10.8
Timezone: America/Los_Angeles

Thread Information:
 => MainThread (alive=True daemon=False)
    Thread-2 (alive=True daemon=True)

Stack Trace:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/gi/overrides/Gtk.py", line 403, in _full_callback
    handler, args = self._extract_handler_and_args(obj_or_map, handler_name)
  File "/usr/lib/python2.7/dist-packages/gi/overrides/Gtk.py", line 376, in _extract_handler_and_args
    raise AttributeError('Handler %s not found' % handler_name)
AttributeError: Handler signal_checkbutton_toggled not found

**Comment by zeroSteiner** [comment]
Thanks @mkayoh for this bug report! I'll get this fixed before dev is moved into master later this week. Nice that someone else is helping test the dev branch.

**Comment by zeroSteiner** [comment]
TLDR just fixed this in commit 77748646779add1301df4e7542dad5ff6aaa2b74.

This seems to have been a bug introduced by the changes in the Glade* class dependencies definition. Specifically the `GladeGObject.dependencies` class attribute was not unique causing the `GladeDependencies.name` member of it to be overwritten under certain conditions such as when both the `HostKeyAcceptDialog` and `HostKeyWarnDialogs` classes were defined with automatic names from the `GladeGObjectMeta`.
